src/fisher.jl

```diff
@@ -24,6 +24,33 @@
 
 export FisherExactTest
 
+"""
+    FisherExactTest(a::Integer, b::Integer, c::Integer, d::Integer)
+
+Perform Fisher's exact test of the null hypothesis that the success probabilities ``a/c``
+and ``b/d`` are equal, that is the odds ratio ``(a/c) / (b/d)`` is one, against the
+alternative hypothesis that they are not equal.
+
+The contingency table is structured as:
+
+| -  | X1 | X2 |
+|:--:|:--:|:--:|
+|*Y1*| a  | b  |
+|*Y2*| c  | d  |
+
+!!! note
+    The [`Base.show`](@ref) output contains the conditional maximum likelihood estimate of the odds ratio
```

Comment:
As noted above

> The Documenter.jl documentation states
"*Note that depending on what the CurrentModule is set to, a docstring @ref may need to be prefixed by the module which defines it.*"
> However, at least in my local build, links like [`StatsBase.counts`](@ref) don't seem to work. But maybe we can figure this out in the next PR that will actually set up Documenter.jl.

So, at least for now the links to outside modules don't work. Those are:

- ``[`StatsBase.counts`](@ref)`` and  `` [`Base.show`](@ref)`` for which the error is that `!! No doc found for reference`
- ``[`Rmath.pwilcox`](@ref)`` and ``[`Rmath.psignrank`](@ref)`` for which it says `ERROR: UndefVarError(:Rmath)`, so this is maybe not the correct module call.

The Documenter.jl documentation unfortunately doesn't give an example how to link to other modules and I also couldn't find an example in other packages.

Besides these dead links, the docs are still built correctly and everything looks ok. If you have an idea on how to fix it, I can push the fix here. If it takes us longer to figure it out, we might want to merge here and then fix the links in the follow-up PR.